wasm2c/wasm-rt-impl.c

```diff
@@ -145,6 +145,11 @@ static void os_cleanup_signal_handler(void) {
 static void* os_mmap(size_t size) {
   int map_prot = PROT_NONE;
   int map_flags = MAP_ANONYMOUS | MAP_PRIVATE;
+#ifdef PROT_MPROTECT
+  /* NetBSD's MPROTECT hardening requries declaring that the memory will
+     later change protections to W/X. */
+  map_prot |= PROT_MPROTECT(PROT_READ | PROT_WRITE | PROT_EXEC);
```

Comment:
Hmm, is `PROT_EXEC` necessary? I think it's only ever going to become `PROT_READ | PROT_WRITE`.

Reply:
Ah yes, you are right... we are not actually doing any JIT'ing in wasm2c, its all ahead of time.